apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj

```diff
@@ -464,13 +464,15 @@
 				"${PODS_ROOT}/GoogleMaps/Maps/Frameworks/GoogleMaps.xcframework/ios-arm64_x86_64-simulator/GoogleMaps.framework/Resources/GoogleMaps.bundle",
 				"${PODS_ROOT}/GooglePlaces/Frameworks/GooglePlaces.xcframework/ios-arm64/GooglePlaces.framework/Resources/GooglePlaces.bundle",
 				"${PODS_ROOT}/GooglePlaces/Frameworks/GooglePlaces.xcframework/ios-arm64_x86_64-simulator/GooglePlaces.framework/Resources/GooglePlaces.bundle",
+				"${PODS_CONFIGURATION_BUILD_DIR}/RCT-Folly/RCT-Folly_privacy.bundle",
```

Comment:
I think these privacy bundles shouldn't be there. Could you remove `Pods` folder and reinstall pods?

Reply:
this ended up getting added here https://github.com/expo/expo/pull/31824/files#diff-6a3e90facd4517d1ac8fc0a8d5bdecd99c5368b5cd3505c67aad7ec5306d1805